tests/onegov/translator_directory/test_views.py

```diff
@@ -72,123 +73,126 @@ def test_view_translator(client):
+            for dl in page.pyquery('dl')
+        }
+        assert len(values) == 24
+        assert values['Personal Nr.'] == '978654'
+        assert values['Zulassung'] == ('nicht akkreditiert / Einsatz '
+                                       'Dringlichkeit')
+        assert values['Quellensteuer'] == 'Nein'
+        assert values['Selbständig'] == 'Nein'
+        assert values['Geschlecht'] == 'Männlich'
+        assert values['PLZ'] == '7890'
+        assert values['Wegberechnung'] == f'{round(drive_distance, 1)} km'
+        assert values['AHV-Nr.'] == '756.1234.5678.97'
+        assert values['IBAN'] == 'DE07 1234 1234 1234 1234 12'
+        assert values['E-Mail'] == 'test@test.com'
+        assert values['Telefon Mobile'] == '079 700 80 97'
+        assert values['Fachkenntnisse nach Dolmetscherart'] == \
+            'Simultandolmetschen'
+        assert 'Ernährung' in values['Fachkenntnisse nach Berufssparte']
+        assert 'Psychologie' in values['Fachkenntnisse nach Berufssparte']
+        assert values['Muttersprachen'] == language_names[3]
+        assert language_names[0] in values['Arbeitssprache - Wort']
+        assert language_names[1] in values['Arbeitssprache - Wort']
+        assert values['Arbeitssprache - Schrift'] == language_names[2]
+        assert values['Referenzen Behörden'] == 'All okay'
+        assert values['Ausbildung Dolmetscher'] == 'Nein'
+        assert values['Versteckt'] == 'Nein'
+        assert values['Zertifikate'] == cert_names[0]
+        assert values['Heimatort'] == 'Gersau'
+        assert values['Nationalität'] == 'CH'
+
+        # test user account created and activation mail sent
+        user = UserCollection(session).by_username('test@test.com')
+        assert user.translator.title == 'Bob, Uncle'
+        assert user.active is True
+        assert user.role == 'translator'
 
-    mail = client.get_email(0, flush_queue=True)
-    assert mail['To'] == 'test@test.com'
-    assert mail['Subject'] == 'Ein Konto wurde für Sie erstellt'
+        mail = client.get_email(0, flush_queue=True)
+        assert mail['To'] == 'test@test.com'
+        assert mail['Subject'] == 'Ein Konto wurde für Sie erstellt'
 
-    # test translator can login and view his own data
-    client.logout()
-    reset_password_url = re.search(
-        r'(http://localhost/auth/reset-password[^)]+)',
-        mail['TextBody']
-    ).group()
-    page = client.get(reset_password_url)
-    page.form['email'] = 'test@test.com'
-    page.form['password'] = 'p@ssw0rd'
-    page.form.submit()
+        # test translator can login and view his own data
+        client.logout()
+        reset_password_url = re.search(
+            r'(http://localhost/auth/reset-password[^)]+)',
+            mail['TextBody']
+        ).group()
+        page = client.get(reset_password_url)
+        page.form['email'] = 'test@test.com'
+        page.form['password'] = 'p@ssw0rd'
+        page.form.submit()
 
```

Comment:
```suggestion
```suggestion
    with freeze_time('2021-12-31'):
        page = client.login('test@test.com', 'p@ssw0rd', None).maybe_follow()
        assert 'Sind ihre Daten noch aktuell? Bitte überprüfen Sie' not in page
        assert '978654' in page
        assert 'Uncle' in page
        assert 'Bob' in page
```
```